src/commissaire_http/handlers/clusters.py

```diff
@@ -25,11 +29,222 @@ def list_clusters(message, bus):
 
     :param message: jsonrpc message structure.
     :type message: dict
+    :param bus: Bus instance.
+    :type bus: commissaire_http.bus.Bus
+    :returns: A jsonrpc structure.
+    :rtype: dict
+    """
+    clusters_msg = bus.request('storage.list', params=['Clusters'])
+    return create_response(
+        message['id'],
+        [cluster['name'] for cluster in clusters_msg['result']])
+
+
+def get_cluster(message, bus):
+    """
+    Lists all clusters.
```

Comment:
Docs for `get_cluster` still needs updating.  Wrong description + missing `:param bus:`.